Libraries/Azure.psm1

```diff
@@ -165,17 +166,8 @@ Function Validate-SubscriptionUsage($RGXMLData, $Location, $OverrideVMSize, $Sto
 
     #region Storage Accounts
     $currentStorageStatus = Get-AzStorageUsage -Location $Location
-    if ( ($premiumVMs -gt 0 ) -and ($StorageAccount -imatch "NewStorage_")) {
-        $requiredStorageAccounts = 1
-    }
-    elseif ( ($premiumVMs -gt 0 ) -and !($StorageAccount -imatch "NewStorage_")) {
-        $requiredStorageAccounts = 1
-    }
-    elseif ( !($premiumVMs -gt 0 ) -and !($StorageAccount -imatch "NewStorage_")) {
-        $requiredStorageAccounts = 0
-    }
     $overFlowErrors += Check-OverflowErrors -ResourceType "Storage Account" -CurrentValue $currentStorageStatus.CurrentValue `
-        -RequiredValue $requiredStorageAccounts -MaximumLimit $currentStorageStatus.Limit -AllowedUsagePercentage $AllowedUsagePercentage
+        -RequiredValue 0 -MaximumLimit $currentStorageStatus.Limit -AllowedUsagePercentage $AllowedUsagePercentage
```

Comment:
I don't understand this change '0'

Reply:
it's useless, already removed that section in the latest update.